content/docs/command-reference/run.md

```diff
@@ -185,7 +186,8 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   reason.
 
 - `--outs-persist <path>` - declare output file or directory that will not be
-  removed upon `dvc repro`.
+  removed when `dvc repro` starts (but it will still be overwritten when it
```

Comment:
this is not correct ... not necessarily overwritten. It can stay the same, it can be appended in case of a dir, etc

Reply:
True. Changing to "but it can still be modified, overwritten, or even deleted by the stage command(s)" in my Git stash for now.

Reply:
I think it's fine to omit this completely. That's expected by default to my mind.

Reply:
I personally had to try this to see what it did. It wasn't obvious what to expect. It's not that clear by just saying "will not be removed". A lot of users are surprised to learn that DVC deletes outputs before repro in the first place.

Reply:
kk, up to you on this one!